content/en/learn.md

```diff
@@ -3,108 +3,85 @@ title: Learn
+* [Introduction to Numerical Computing with NumPy](http://youtu.be/ZB7BZMhfPgk), Alex Chabot-Leclerc, presented at SciPy 2019.` 
 
-[Educator](github.com/numpy/numpy.org/blob/master/content/en/learn.md#EDUCATOR)
+***
 
-<a name="#BEGINNER"><h2>BEGINNER</h2></a>
+## Advanced 
 
-<h3>Books</h3>
+If you have the basic know-how of NumPy and how to use it, you can try these advanced tutorials, books and videos for a better understanding of specific NumPy concepts such as advanced indexing, splitting, stacking, linear algebra and more.
 
-The[Goodreads list](https://www.goodreads.com/shelf/show/python-scipy) of books on the subject of "Python+SciPy". Most of the books on this list are about the "SciPy ecosystem", which has NumPy at its core.
+<i class="fad fa-chalkboard"></i> **Tutorials**
 
-</br>
+* [100 NumPy Exercises](http://www.labri.fr/perso/nrougier/teaching/numpy.100/index.html), Nicolas P. Rougier, 2016.
+* [An Introduction to NumPy and Scipy](https://engineering.ucsb.edu/~shell/che210d/numpy.pdf), M. Scott Shell, 2014. 
+* [Numpy Medkits](http://mentat.za.net/numpy/numpy_advanced_slides/), Stéfan van der Walt, 2008.
+* [NumPy in Python (Advanced)](https://www.geeksforgeeks.org/numpy-python-set-2-advanced/)
+* [Advanced Indexing](https://www.tutorialspoint.com/numpy/numpy_advanced_indexing.htm)
+* [Machine Learning and Data Analytics with NumPy](https://www.machinelearningplus.com/python/numpy-tutorial-python-part2/)
 
-<h3>Tutorials</h3>
+<i class="fas fa-books"></i> **Books**
 
-[NumPy tutorial](https://numpy.org/devdocs/user/quickstart.html)
+* [NumPy User Guide](https://numpy.org/devdocs/user/index.html)
+* [NumPy Developer Guide](https://numpy.org/devdocs/dev/index.html)
+* [NumPy Reference](https://numpy.org/devdocs/reference/index.html) 
 
-[SciPy Lectures](https://scipy-lectures.org/) (intro for a broader introduction to the scientific Python ecosystem)
+<i class="far fa-file-video"></i> **Videos**
 
-[Towards Data Science](https://towardsdatascience.com/the-ultimate-beginners-guide-to-numpy-f5a2f99aef54), Anne Bonner.
+* [Scientific Computing with Python - SciPy Japan, 2019](https://www.youtube.com/watch?v=cYugp9IN1-Q)
+* [Advanced Indexing operations in NumPy arrays](https://www.youtube.com/watch?v=2WTDrSkQBng) 
+* [Machine learning in Trading - Georgia Tech Udacity course](https://www.udacity.com/course/machine-learning-for-trading--ud501)
 
-[Guru99](https://www.guru99.com/numpy-tutorial.html)
+***
 
-[Machine Learning Plus](https://www.machinelearningplus.com/python/numpy-tutorial-part1-array-python-examples/)
+## Citing NumPy
 
-[Edureka](https://www.edureka.co/blog/python-numpy-tutorial/)
+Here is a Bib Tex citation that can be used to cite NumPy:
 
-[Dataquest](https://www.dataquest.io/blog/numpy-tutorial-python/)
+    @book{oliphant2006guide, 
+      title={A guide to NumPy}, 
+      author={Oliphant, Travis E}, 
+      volume={1}, 
+      year={2006}, 
+      publisher={Trelgol Publishing USA} 
+    }
```

Comment:
Could you add both citations as on https://scipy.org/citing.html#numpy?

Reply:
done